.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Before submitting
+
+Before submitting a PR, please make sure to check our contributing guidelines regarding [code formatting](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#code-formatting-and-typing), [tests](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#unit-tests), and [documentation](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#documentation).
+
+## What does this PR do?
+
+Fixes # (issue).
+
+## PR review
+
+Anyone in the community is free to review the PR once the tests have passed.
```

Comment:
I'd remove that too, that's not really useful info for the PR author. Also we often don't wait for the tests lol